tests/AnonymizerTest.php

```diff
@@ -285,12 +283,18 @@ public function testRun(): void
         $this->assertSame(['INSERT table1' . "\n", 'INSERT table2'], $actual);
     }
 
-    private function makeLineStream(array $lines)
+    private function makeInputStream(array $lines)
     {
+        $fp = fopen('data://text/plain;base64,' . base64_encode(implode("\n", $lines)), 'ab+');
+        rewind($fp);
 
-        $fp = fopen(self::TMP_FILE_PATH . (string)rand(0, 999), 'w+');
-        fwrite($fp, implode("\n", $lines));
+        return $fp;
+    }
 
+    private function makeOutputStream(array $lines)
```

Comment:
No need for a parameter on `makeOutputStream()`. It should be just an empty writable stream.